react-basic-practice/src/pages/dashboard.tsx

```diff
@@ -36,6 +44,12 @@ const Dashboard: React.FC = () => {
       );
   };
 
+  const renderModalContent = () =>
+    modalTitle === MODAL_TITLES.VIEW_CUSTOMER && viewData ? (
+      <CustomerView data={viewData} />
+    ) : (
+      <CustomerForm />
```

Comment:
how about Edit? It's related to previous PR

Reply:
because edit and add use the same form